src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1581,44 +1581,6 @@ def test_acc_parallel_invalid_node():
             "ACCParallelTrans transformation" in str(err))
 
 
-def test_acc_data_copyin(tmpdir):
-    ''' Test that we correctly generate the arguments to the copyin
-    clause of an OpenACC data region '''
-    psy, invoke = get_invoke("single_invoke_three_kernels.f90", API, idx=0)
-    schedule = invoke.schedule
-
-    accpt = ACCParallelTrans()
-    accdt = ACCEnterDataTrans()
-
-    # Put each loop within an OpenACC parallel region
-    for child in schedule.children:
-        if isinstance(child, Loop):
-            new_sched, _ = accpt.apply(child)
-
-    # Create a data region for the whole schedule
-    new_sched, _ = accdt.apply(new_sched)
-
-    invoke.schedule = new_sched
-    code = str(psy.gen)
-
-    # Check that we've correctly declared the logical variable that
-    # records whether this is the first time we've entered this invoke.
-    assert "LOGICAL, save :: first_time=.True." in code
-
-    pcopy = (
-        "      IF (first_time) THEN\n"
```

Comment:
I know we no longer have the IF(first_time)THEN bit but it seems a bit strong to delete the whole test? Is the functionality being tested now in psyGen?

Reply:
You're right, I was over zealous. I've reinstated the test with the appropriate part.